.github/PULL_REQUEST_TEMPLATE/package_addition.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Package addition
```

Comment:
This could also be for a package upgrade. My one request would be that you update wording to include upgrades as well as new packages.

Reply:
Good point. I'll add it.